layouts/partials/menu.html

```diff
@@ -1,11 +1,12 @@
 <ul>
-	{{ range . }}
-	<li>
+	{{ range $.menu }}
+	<li{{if $.node.HasMenuCurrent "main" . }} class="parent-active"{{end}}{{if $.node.IsMenuCurrent "main" . }} class="active"{{end}}>
```

Comment:
Do we really need `parent-active` or is the following selector enough:
```css
li ul > li.active { }
```
And if we should have it, is the name `active-parent` better?